lib/pengine/unpack.c

```diff
@@ -2287,9 +2294,9 @@ process_orphan_resource(const xmlNode *rsc_entry, const pcmk_node_t *node,
 
     } else {
         CRM_CHECK(rsc != NULL, return NULL);
-        pcmk__rsc_trace(rsc, "Added orphan %s", rsc->id);
+        pcmk__rsc_trace(rsc, "Added \"removed\" resource %s", rsc->id);
         resource_location(rsc, NULL, -PCMK_SCORE_INFINITY,
-                          "__orphan_do_not_run__", scheduler);
+                          "__removed_do_not_run__", scheduler);
```

Comment:
This is a little odd.  I searched around and this is the only reference to this string.  It's definitely not part of any public facing anything, right?